jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/SavedModelBundleExecutor.scala

```diff
@@ -19,23 +19,22 @@ import org.tensorflow.example.SequenceExample
   * @param config
   * @see <a href="https://www.tensorflow.org/api_docs/java/reference/org/tensorflow/SavedModelBundle">SavedModelBundle</a>
   */
-class SavedModelBundleExecutor(dirPath: String, config: ModelExecutorConfig)
-    extends (SequenceExample => Array[Float]) {
+class SavedModelBundleExecutor(dirPath: String, config: ModelExecutorConfig) extends (SequenceExample => Array[Float]) {
   val savedModelBundle = SavedModelBundle.load(dirPath, "serve")
   val session = savedModelBundle.session()
 
   override def apply(proto: SequenceExample): Array[Float] = {
-    val ModelExecutorConfig(inputNode, outputNode, padTo, _) = config
+    val ModelExecutorConfig(inputNode, outputNode, _, _) = config
```

Comment:
Let's remove the 3rd and 4th args from ModelExecutorConfig? Currently, I manually set it to 50 for num docs per query and 20 as max query len. But since we are not using this anymore, we should just remove to simplify the client

Reply:
Good point! Will do that.